app/models/like.rb

```diff
@@ -1,4 +1,5 @@
 class Like < ApplicationRecord
   belongs_to :user
   belongs_to :post
+  scope :update_likes_counter, ->(post, count) { post.update(likes_counter: count) }
```

Comment:
- [ ] According to the guidelines you should have a method that updates the likes counter for a post. You can check out the guide [Scope vs Class Methods](https://www.square63.com/scope-vs-class-methods/)

```ruby
def update_likes_counter
  post.increment!(:likes_counter)
end
```